04.ls/ls.rb

```diff
@@ -2,28 +2,51 @@
 # frozen_string_literal: true
 
 require 'optparse'
+require 'etc'
 
 INDENT_SIZE = 2
 MAX_NUM_OF_COLUMNS = 3
 WINDOW_WIDTH = `tput cols`.chomp.to_i
+BLOCK_SIZE_RATIO = 2 # Rubyとos標準lsコマンド間のブロックサイズの基準の違いを補正する比率
+
+FILE_TYPES = {
+  1 => 'p',
+  2 => 'c',
+  4 => 'd',
+  6 => 'b',
+  10 => '-',
+  12 => 'l',
+  14 => 's'
+}.freeze
+PERMISSION_TYPES = {
```

Comment:
ちょっとしたことですが、空行開けた方が見やすいと思います。

```suggestion
}.freeze

PERMISSION_TYPES = {
```